### PR TITLE
[FIX] account_followup: "Print overdue payments" prints non-overdue paym...

### PR DESCRIPTION
**Pull request by simongoffin**
...ents

"Print overdue payments" must  print all overdue payements and not non-overdue.

opw:628874

### CODE REVIEW

File: addons/account_followup/account_followup.py [hunk 1]
```diff
@@ -309,6 +310,7 @@ def do_button_print(self, cr, uid, ids, context=None):
                                                                    ('reconcile_id', '=', False),
                                                                    ('state', '!=', 'draft'),
                                                                    ('company_id', '=', company_id),
+                                                                   ('date_maturity', '<=', date.today().strftime('%Y-%m-%d')),
```
Comment (mart-e) [inline addons/account_followup/account_followup.py]: use `fields.date.context_today()` instead.

**Comment by beledouxdenis** [comment]
This issue doesn't happen in 7.0 ?
Replace your commit message by
>[FIX] account_followup: overdue payments must exclude payments not yet due. 

>Before this revision, the overdue payments report contained all payments due,
including the ones in the future, that are not actually due yet. 

>opw-628874

Other than that, if your fix actually works, it's fine for me. Please double-check if this fix should not be applied in 7.0 :) !

**Comment by beledouxdenis** [comment]
If you tested your fix, I am okay with it :). You can merge in stable 7.0.

**Comment by beledouxdenis** [comment]
Closed with ab9f02cdee91d9caa873f190a40bbb17e6cbc7b4

**Pull request closed without merging**